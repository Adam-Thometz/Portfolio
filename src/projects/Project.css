.Project {
  width: 85%;
  margin: 50px auto;
  border-radius: 20px;
  padding: 20px;
}

.Project h2 {
  text-align: center;
}

.Project-info-img-container {
  display: flex;
}

.Project-award {
  height: 50px;
  width: 50px;
}

.Project-info {
 width: 50%;
 text-align: left;
 padding: 20px;
}

.Project-links a {
  font-size: 2.5rem;
  padding: 10px;
}

.demo {
  color: rgb(87, 177, 250);
}

.github {
  color: black;
}

.Project-img {
  width: 40%;
  padding: 20px;
  margin: auto;
}

.Project-img img {
  height: 70%;
  width: 100%;
  border-radius: 10px;
  border: 2px solid;
  box-shadow: 0px 0px 50px 5px;
}

.reptifeed {
  background-color: rgba(69, 153, 0, 0.35);
  box-shadow: 0px 0px 20px 15px rgba(69, 153, 0, 0.5),
    inset 0px 0px 10px 5px rgba(69, 153, 0, 0.5);
}

.reptifeed img {
  border-color: rgba(154, 177, 6);
}

.eight-bit-clan {
  background-color: rgba(250, 250, 250, 0.35);
  color: rgb(12, 17, 88);
  box-shadow: 0px 0px 20px 15px rgba(250, 250, 250, 0.5),
    inset 0px 0px 10px 5px rgba(250, 250, 250, 0.5);
}
  
.eight-bit-clan img {
  border-color: rgb(37, 53, 113);
}
  
.catch-your-feelings {
  background-color: rgba(156, 156, 201, 0.5);
  box-shadow: 0px 0px 20px 15px rgb(156, 156, 201, 0.5),
    inset 0px 0px 10px 5px rgb(156, 156, 201, 0.5);
}
  
.catch-your-feelings img {
  border-color: rgb(56, 44, 44);
}
    
.musophile {
  background-color: rgba(98, 187, 255, 0.35);
  box-shadow: 0px 0px 20px 15px rgba(98, 187, 255, 0.5),
    inset 0px 0px 10px 5px rgba(98, 187, 255, 0.5);
}

.musophile img {
  border-color: 2px solid rgb(182, 237, 255);
}

@media (max-width: 1200px) {
  .Project-info-img-container {
    flex-direction: column-reverse;
  }

  .Project-info {
    width: 96%;
    text-align: center;
  }
  
  .Project-info li {
    list-style-type: none;
    padding: 0;
    margin: 0;
  }

  .Project-img {
    width: 80%;
  }
}