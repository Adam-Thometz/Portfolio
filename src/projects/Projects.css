.Projects {
  margin: 32px auto;
  animation: slide-up var(--duration) calc(var(--delay) * var(--i)) ease-in-out
    both;
  text-align: center;
  display: flex;
  flex-direction: column;
  justify-content: space-evenly;
  transition: opacity 0.3s ease-in-out;
  position: relative;
}

.Projects-project-list {
  display: flex;
  flex-wrap: wrap;
  justify-content: center;
  margin: 0 auto;
}

.project {
  transition: opacity 0.3s ease-in-out;
}

.display {
  transition: opacity 0.3s ease-in-out;
}

/* Styles for projects */

.art-buddy {
  background-color: rgba(77, 79, 94, 0.35);
  color: rgb(137, 235, 10);
  box-shadow: 0px 0px 20px 15px rgba(253, 77, 58, 0.5);
}

.art-buddy .Project-img img {
  border-color: rgb(255, 232, 26);
}

.reptifeed {
  background-color: rgba(69, 153, 0, 0.35);
  color: #d5ce3b;
  box-shadow: 0px 0px 20px 15px rgba(69, 153, 0, 0.5);
}

.eight-bit-clan {
  color: rgba(250, 250, 250);
  background-color: rgb(12, 17, 88, 0.35);
  box-shadow: 0px 0px 20px 15px rgba(250, 250, 250, 0.5);
}

.vesta-healing-psychotherapy {
  background-color: rgba(203, 155, 255, 0.35);
  color: rgba(255, 197, 162, 1);
  box-shadow: 0px 0px 20px 15px rgba(245, 129, 139, 0.5);
}

@media (max-width: 550px) {
  .Projects {
    flex-direction: column;
    align-items: center;
  }
}
