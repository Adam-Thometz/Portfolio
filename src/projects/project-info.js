// Project pictures
import artBuddy from './screenshots/art-buddy.png';
import reptifeed from './screenshots/reptifeed.png';
import bitClan from './screenshots/8-bit-clan.png';
import catchYourFeelings from './screenshots/catch-your-feelings.png';
import musophile from './screenshots/musophile.png';

// Awards
import mintbeanStar from './screenshots/mintbean-star.gif';

const projects = [
  {
    name: 'Art Buddy',
    description: 'First "real-world" project since graduating from bootcamp. Art Buddy is a collection of arts-based games and tools designed specifically for the special education classroom and reaching student academic goals. Games are based on real lessons I taught during my time as a special education music teacher. Art Buddy was made in collaboration with a UX designer and fellow educator. This game is currently in progress. For now, the only working games are Word-To-Music Decoder, Sequence Maker, and Score Keeper. There are remnants of Whats That Instrument currently deployed.',
    techStack: ["Javascript", "React", "CSS", "Redux", "Jest"],
    img: artBuddy,
    links: {
      demo: 'https://art-buddy.surge.sh',
      github: 'https://github.com/Adam-Thometz/Art-Buddy'
    }
  },
  {
    name: 'Reptifeed',
    description: 'My second and final capstone project for Springboard. Inspired by my new baby blue-tongue skink, Unity, this app gives you ideas on what to feed your reptile and helps you keep track of the foods you have for them. Features a homemade API for food information.',
    techStack: ["Javascript", "Node", "Express", "React", "CSS", "SQL", "Postgres", "Jest"],
    login: {
      username: 'abc',
      password: '123'
    },
    img: reptifeed,
    links: {
      demo: 'https://reptifeed.surge.sh',
      github: 'https://github.com/Adam-Thometz/Reptifeed'
    }
  },
  {
    name: '8-Bit Clan',
    award: mintbeanStar,
    description: "A pixel art maker created for the Mintbean Hiring Hackathon, Nov 2021. Create art, save it to your profile, and see if it made it to the trending list! Built in a week with UX designer Melody Acosta. Winner of Mintbean Star, awarded to top 5% of competitors.",
    techStack: ["Javascript", "Node", "Express", "React", "JSS", "SQL", "Postgres", "Heroku"],
    login: {
      username: 'abc',
      password: '123'
    },
    img: bitClan,
    links: {
      demo: 'https://8-bit-clan.surge.sh/',
      github: 'https://github.com/Adam-Thometz/8-Bit-Clan'
    }
  },
  {
    name: 'Catch Your Feelings',
    description: "An app that I invented for my day job as a music teacher for autistic students in grades PreK-5. You describe how happy and energetic you feel using icons and you get a song back based on the input. This project is currently being revamped!",
    techStack: ["Python", "Flask", "HTML", "CSS", "Javascript", "Heroku"],
    img: catchYourFeelings,
    links: {
      demo: 'https://catch-your-feelings.herokuapp.com/',
      github: 'https://github.com/Adam-Thometz/Catch-Your-Feelings/tree/flask'
    }
  },
  // {
  //   name: "Colorful Melodies",
  //   description: "An app that takes words and turns them into a simple melody by having students identify letters. Used as a tool in my music classroom to teach musical pitch by associating pitches with colors and letters. Created with the goal of incorporating letter recognition into mutlisensory musical instruction for students with learning and sensory-based disabilities. Uses the Web Audio API.",
  //   techStack: ["HTML", "CSS", "Javascript"],
  //   img: musicColors,
  //   links: {
  //     demo: '',
  //     github: ''
  //   }
  // },
  {
    name: 'Musophile',
    description: 'My first capstone project for Springboard. A website where you can curate your own music library. Created using the Spotify and MusicBrainz APIs.',
    techStack: ["Python", "Flask", "Postgres", "HTML", "CSS", "Javascript", "jQuery", "Heroku"],
    login: {
      username: 'abc',
      password: '12345678'
    },
    img: musophile,
    links: {
      demo: 'https://musophile.herokuapp.com/',
      github: 'https://github.com/Adam-Thometz/Musophile'
    }
  },
];

export default projects;