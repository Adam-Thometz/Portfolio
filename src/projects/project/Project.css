.Project {
  width: 85%;
  margin: 50px auto;
  border-radius: 20px;
  padding: 20px;
  position: relative;
}

.Project-description a {
  color: white;
}

.Project h2 {
  text-align: left;
}

.Project-info-img-container {
  display: flex;
  align-items: top;
}

.Project-award {
  height: 50px;
  width: 50px;
}

.Project-info {
  width: 50%;
  text-align: left;
  padding: 20px;
}

.Project-links a {
  font-size: 2.5rem;
  padding: 10px;
  transition: font-size .5s ease-in-out;
  transition: background-color .5s ease-in-out;
}

.Project-links a:hover {
  background-color: rgb(125, 142, 158);
}

.demo {
  color: rgb(87, 177, 250);
}

.github {
  color: black;
}

.Project-img {
  width: 40%;
  padding: 20px;
  margin: auto;
}

.Project-img img {
  height: 70%;
  width: 100%;
  border-radius: 10px;
  border: 2px solid;
  box-shadow: 0px 0px 50px 5px;
}

.art-buddy {
  background-color: rgba(77, 79, 94, 0.75);
  box-shadow: 0px 0px 20px 15px rgba(137, 235, 10, 0.5),
    inset 0px 0px 10px 5px rgba(137, 235, 10, 0.5);
}

.art-buddy img {
  border-color: rgb(255, 232, 26);
}

.reptifeed {
  background-color: rgba(69, 153, 0, 0.35);
  box-shadow: 0px 0px 20px 15px rgba(69, 153, 0, 0.5),
    inset 0px 0px 10px 5px rgba(69, 153, 0, 0.5);
}

.reptifeed img {
  border-color: rgba(154, 177, 6);
}

.eight-bit-clan {
  background-color: rgba(250, 250, 250, 0.35);
  color: rgb(12, 17, 88);
  box-shadow: 0px 0px 20px 15px rgba(250, 250, 250, 0.5),
    inset 0px 0px 10px 5px rgba(250, 250, 250, 0.5);
}
  
.eight-bit-clan img {
  border-color: rgb(37, 53, 113);
}
  
.catch-your-feelings {
  background-color: rgba(156, 156, 201, 0.5);
  box-shadow: 0px 0px 20px 15px rgb(156, 156, 201, 0.5),
    inset 0px 0px 10px 5px rgb(156, 156, 201, 0.5);
}
  
.catch-your-feelings img {
  border-color: rgb(56, 44, 44);
}
    
.musophile {
  background-color: rgba(98, 187, 255, 0.35);
  box-shadow: 0px 0px 20px 15px rgba(98, 187, 255, 0.5),
    inset 0px 0px 10px 5px rgba(98, 187, 255, 0.5);
}

.musophile img {
  border-color: 2px solid rgb(182, 237, 255);
}

@media (max-width: 1200px) {
  .Project-info-img-container {
    flex-direction: column;
  }

  .Project-info {
    width: 85%;
    margin: 0 auto;
    text-align: left;
  }

  .Project-img {
    width: 80%;
  }

  .Project-links {
    top: 0;
  }

  .Project-links a {
    font-size: 1.5rem;
    padding: 10px;
  }
}