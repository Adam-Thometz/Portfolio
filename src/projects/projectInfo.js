// Project pictures
import artBuddy from '../_media/imgs/art-buddy.webp';
import reptifeed from '../_media/imgs/reptifeed.webp';
import bitClan from '../_media/imgs/8-bit-clan.webp';
import musophile from '../_media/imgs/musophile.webp';

// Awards
import mintbeanStar from '../_media/imgs/mintbean-star.gif';

const projects = [
  {
    name: 'Art Buddy',
    shortDescription: 'Arts-based games for special education',
    description: <><p>Art Buddy is a collection of arts-based games and tools designed specifically for the special education classroom and reaching students' academic goals. I worked with UX designer/fellow educator <a href='https://uxfol.io/mattiebecker'>Mattie Becker</a> to make this.</p><b>NOTE: I recommend viewing on a desktop. This app was purposely designed for a large screen.</b></>,
    learnedHowTo: [
      'make my code readable, obvious, and testable (I call it "making my code ROT")',
      'use package.json scripts to minimize typing in the terminal',
      'make music play in the browser depending on app settings',
      'create a component library of common UI elements',
      'articulate my vision for a product',
      'incorporate user feedback',
    ],
    techStack: ["Javascript", "React", "CSS", "Redux", "Jest"],
    img: artBuddy,
    links: {
      demo: 'https://art-buddy.surge.sh',
      github: 'https://github.com/Adam-Thometz/Art-Buddy'
    }
  },
  {
    name: 'Reptifeed',
    shortDescription: 'A reptilian diet tracker',
    description: <p>My second and final capstone project for Springboard. Inspired by my blue-tongue skink, Unity, this app gives you ideas on what to feed your reptile and helps you keep track of the foods you have for them.</p>,
    learnedHowTo: [
      'build admin-only features',
      'make a website/app accessible',
      'make a responsive website',
      'invent my own API'
    ],
    techStack: ["Javascript", "Node", "Express", "React", "CSS", "Postgres", "Jest"],
    login: {
      username: 'abc',
      password: '123'
    },
    img: reptifeed,
    links: {
      demo: 'https://reptifeed.surge.sh',
      github: 'https://github.com/Adam-Thometz/Reptifeed'
    }
  },
  {
    name: '8-Bit Clan',
    award: mintbeanStar,
    shortDescription: 'An 8-bit NFT maker',
    description: <p>A pixel art maker created for the Mintbean Hiring Hackathon, Nov 2021. Create art, save it to your profile, and see if it made it to the trending list! Built with UX designer <a href='https://melodyux.design/'>Melody Acosta</a> in a week. Winner of Mintbean Star, awarded to top 5% of competitors.</p>,
    learnedHowTo: [
      'collaborate with a UX designer',
      'work efficiently under tight time constraints',
      'store art data in a database and call on that data to reproduce a smaller version of that art'
    ],
    techStack: ["Javascript", "Node", "Express", "React", "JSS", "Postgres"],
    login: {
      username: 'abc',
      password: '123'
    },
    img: bitClan,
    links: {
      demo: 'https://8-bit-clan.surge.sh/',
      github: 'https://github.com/Adam-Thometz/8-Bit-Clan'
    }
  },
  {
    name: 'Musophile',
    shortDescription: 'An obscure music curator',
    description: <p>My first capstone project for Springboard. A website where you can curate your own music library. Created using the Spotify and MusicBrainz APIs.</p>,
    learnedHowTo: [
      'connect frontend and backend using server-side rendering',
      'use the Spotify API and OAuth'
    ],
    techStack: ["Python", "Flask", "Postgres", "HTML", "CSS", "Javascript", "jQuery", "Bootstrap"],
    login: {
      username: 'abc',
      password: '12345678'
    },
    img: musophile,
    links: {
      demo: 'https://musophile.herokuapp.com/',
      github: 'https://github.com/Adam-Thometz/Musophile'
    }
  },
];

export default projects