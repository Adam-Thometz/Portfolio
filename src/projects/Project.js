import React from "react";
import { Link } from "react-router-dom";
import TechStack from "../tech-stack/TechStack";
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import { faGlobe } from "@fortawesome/free-solid-svg-icons";
import { faGithub } from "@fortawesome/free-brands-svg-icons"
import './Project.css';

const Project = ({name, award, description, techStack, login, img, links}) => {
  const cssClass = name.toLowerCase().replaceAll(' ', '-').replace('8', 'eight');

  return (
    <div className={`Project ${cssClass}`} key={cssClass}>
      <h2>{name}</h2>
      <div className="Project-info-img-container">
        <div className="Project-info" tabIndex={0}>
          <div className="Project-description">
            <h3>Description</h3>
            <p>{description}</p>
            {award ? <img className="Project-award" src={award} alt={`An award for ${name}`} /> : null}
          </div>
          <div className="Project-links">
            <Link className="demo" to="" onClick={() => window.open(links.demo)}><FontAwesomeIcon title={`Demo for ${name}`} icon={faGlobe} /></Link>
            <Link className="github" to="" onClick={() => window.open(links.github)}><FontAwesomeIcon title={`Github repo for ${name}`} icon={faGithub} /></Link>

            {login ? (
              <div>
                <b>To get the most out of this site, log in with the following:</b>
                <ul>
                  <li>Username: {login.username}</li>
                  <li>Password: {login.password}</li>
                </ul>
              </div>
            ) : null}
          </div>
        </div>
        <div className="Project-img">
          <img src={img} alt={`A screenshot of ${name}`} />
        </div>
      </div>
      <TechStack techStack={techStack} />
    </div>
  );
};

export default Project;