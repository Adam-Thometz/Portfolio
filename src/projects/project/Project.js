import React from "react";
import { Link } from "react-router-dom";
import TechStack from "../tech-stack/TechStack";
import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
import { faGlobe } from "@fortawesome/free-solid-svg-icons";
import { faGithub } from "@fortawesome/free-brands-svg-icons"
import './Project.css';

const Project = ({name, award, description, learnedHowTo, techStack, login, img, links}) => {
  const cssClass = name.toLowerCase().replaceAll(' ', '-').replace('8', 'eight');

  return (
    <div className={`Project ${cssClass}`} key={cssClass}>
      <h2>{name}</h2>
      <div className="Project-info-img-container">
        <div className="Project-info" tabIndex={0}>
          <div className="Project-description">
            <h3>Description</h3>
            {description}
            {learnedHowTo ? <>
              <h3>Building this taught me how to...</h3>
              <ul>
                {learnedHowTo.map(fact => <li>{fact}</li>)}
              </ul>
            </> : null}
            {award ? <img className="Project-award" src={award} alt={`An award for ${name}`} /> : null}
          </div>
            {login ? (
              <div className="Project-login-info">
                <b>To fully explore the app, make your own account or log in with the following:</b>
                <ul>
                  <li>Username: {login.username}</li>
                  <li>Password: {login.password}</li>
                </ul>
              </div>
            ) : null}
          <div className="Project-links">
            {links.demo ? <Link className="demo" to="" onClick={() => window.open(links.demo)}><FontAwesomeIcon title={`Demo for ${name}`} icon={faGlobe} /></Link> : null}
            <Link className="github" to="" onClick={() => window.open(links.github)}><FontAwesomeIcon title={`Github repo for ${name}`} icon={faGithub} /></Link>
          </div>
        </div>
        <div className="Project-img">
          <img src={img} alt={`A screenshot of ${name}`} />
        </div>
      </div>
      <TechStack techStack={techStack} />
    </div>
  );
};

export default Project;