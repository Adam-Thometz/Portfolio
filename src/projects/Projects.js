import React from "react";
import projects from "./project-info";
import Project from "./Project";
import NavContact from "../navbar-contact/NavContact";
import './Projects.css'

const Projects = () => {
  return (
    <main className="Projects">
      <h1 className="Projects-header">Projects</h1>
      {projects.map(p => (
        <Project name={p.name} award={p.award} description={p.description} techStack={p.techStack} login={p.login} img={p.img} links={p.links} />
        ))}
      <NavContact />
    </main>
  );
};

export default Projects;