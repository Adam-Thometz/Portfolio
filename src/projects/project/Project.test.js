import React from "react";
import { render } from "@testing-library/react";
import Project from './Project';

import projects from '../projectInfo';

describe('Project component', () => {
  it('renders without crashing', () => {
    render(<Project project={projects[0]} />);
  });

  it('matches the snapshot', () => {
    const { asFragment } = render(<Project project={projects[0]} />);
    expect(asFragment()).toMatchSnapshot();
  });
});