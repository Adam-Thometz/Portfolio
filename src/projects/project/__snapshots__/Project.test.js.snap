// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Project component matches the snapshot 1`] = `
<DocumentFragment>
  <section
    class="Project art-buddy"
  >
    <div
      class="Project-nav"
    >
      <button
        id="prev"
        title="Previous"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-arrow-left fa-w-14 "
          data-icon="arrow-left"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 448 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M257.5 445.1l-22.2 22.2c-9.4 9.4-24.6 9.4-33.9 0L7 273c-9.4-9.4-9.4-24.6 0-33.9L201.4 44.7c9.4-9.4 24.6-9.4 33.9 0l22.2 22.2c9.5 9.5 9.3 25-.4 34.3L136.6 216H424c13.3 0 24 10.7 24 24v32c0 13.3-10.7 24-24 24H136.6l120.5 114.8c9.8 9.3 10 24.8.4 34.3z"
            fill="currentColor"
          />
        </svg>
      </button>
      <button
        class="Project-exit"
      >
        Back to projects
      </button>
      <button
        id="next"
        title="Next"
      >
        <svg
          aria-hidden="true"
          class="svg-inline--fa fa-arrow-right fa-w-14 "
          data-icon="arrow-right"
          data-prefix="fas"
          focusable="false"
          role="img"
          viewBox="0 0 448 512"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M190.5 66.9l22.2-22.2c9.4-9.4 24.6-9.4 33.9 0L441 239c9.4 9.4 9.4 24.6 0 33.9L246.6 467.3c-9.4 9.4-24.6 9.4-33.9 0l-22.2-22.2c-9.5-9.5-9.3-25 .4-34.3L311.4 296H24c-13.3 0-24-10.7-24-24v-32c0-13.3 10.7-24 24-24h287.4L190.9 101.2c-9.8-9.3-10-24.8-.4-34.3z"
            fill="currentColor"
          />
        </svg>
      </button>
    </div>
    <h1>
      Art Buddy
    </h1>
    <div
      class="Project-info-img-container"
    >
      <div
        class="Project-img"
      >
        <img
          alt="A screenshot of Art Buddy"
          src="art-buddy.webp"
        />
        <section
          class="TechStack"
        >
          <figure
            class="TechStackCard"
          >
            <img
              alt="Javascript"
              class="TechStackCard-img"
              src="javascript.png"
            />
            <figcaption
              class="TechStackCard-caption"
            >
              Javascript
            </figcaption>
          </figure>
          <figure
            class="TechStackCard"
          >
            <img
              alt="React"
              class="TechStackCard-img"
              src="reactJS.png"
            />
            <figcaption
              class="TechStackCard-caption"
            >
              React
            </figcaption>
          </figure>
          <figure
            class="TechStackCard"
          >
            <img
              alt="CSS"
              class="TechStackCard-img"
              src="css.png"
            />
            <figcaption
              class="TechStackCard-caption"
            >
              CSS
            </figcaption>
          </figure>
          <figure
            class="TechStackCard"
          >
            <img
              alt="Redux"
              class="TechStackCard-img"
              src="redux.png"
            />
            <figcaption
              class="TechStackCard-caption"
            >
              Redux
            </figcaption>
          </figure>
          <figure
            class="TechStackCard"
          >
            <img
              alt="Jest"
              class="TechStackCard-img"
              src="jest.png"
            />
            <figcaption
              class="TechStackCard-caption"
            >
              Jest
            </figcaption>
          </figure>
        </section>
      </div>
      <div
        class="Project-info"
        tabindex="0"
      >
        <div
          class="Project-description"
        >
          <h2>
            Description
          </h2>
          <p>
            Art Buddy is a collection of arts-based games and tools designed specifically for the special education classroom and reaching students' academic goals. I worked with UX designer/fellow educator 
            <a
              href="https://uxfol.io/mattiebecker"
            >
              Mattie Becker
            </a>
             to make this.
          </p>
          <b>
            NOTE: I recommend viewing on a desktop. This app was purposely designed for a large screen.
          </b>
          <h2>
            Building this taught me how to...
          </h2>
          <ul>
            <li>
              make my code readable, obvious, and testable (I call it "making my code ROT")
            </li>
            <li>
              use package.json scripts to minimize typing in the terminal
            </li>
            <li>
              make music play in the browser depending on app settings
            </li>
            <li>
              create a component library of common UI elements
            </li>
            <li>
              articulate my vision for a product
            </li>
            <li>
              incorporate user feedback
            </li>
          </ul>
        </div>
        <div
          class="Project-links"
        >
          <a
            class="demo"
            href="https://art-buddy.surge.sh"
            rel="noreferrer"
            target="_blank"
            title="Demo for Art Buddy"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-globe fa-w-16 "
              data-icon="globe"
              data-prefix="fas"
              focusable="false"
              role="img"
              viewBox="0 0 496 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M336.5 160C322 70.7 287.8 8 248 8s-74 62.7-88.5 152h177zM152 256c0 22.2 1.2 43.5 3.3 64h185.3c2.1-20.5 3.3-41.8 3.3-64s-1.2-43.5-3.3-64H155.3c-2.1 20.5-3.3 41.8-3.3 64zm324.7-96c-28.6-67.9-86.5-120.4-158-141.6 24.4 33.8 41.2 84.7 50 141.6h108zM177.2 18.4C105.8 39.6 47.8 92.1 19.3 160h108c8.7-56.9 25.5-107.8 49.9-141.6zM487.4 192H372.7c2.1 21 3.3 42.5 3.3 64s-1.2 43-3.3 64h114.6c5.5-20.5 8.6-41.8 8.6-64s-3.1-43.5-8.5-64zM120 256c0-21.5 1.2-43 3.3-64H8.6C3.2 212.5 0 233.8 0 256s3.2 43.5 8.6 64h114.6c-2-21-3.2-42.5-3.2-64zm39.5 96c14.5 89.3 48.7 152 88.5 152s74-62.7 88.5-152h-177zm159.3 141.6c71.4-21.2 129.4-73.7 158-141.6h-108c-8.8 56.9-25.6 107.8-50 141.6zM19.3 352c28.6 67.9 86.5 120.4 158 141.6-24.4-33.8-41.2-84.7-50-141.6h-108z"
                fill="currentColor"
              />
            </svg>
          </a>
          <a
            class="github"
            href="https://github.com/Adam-Thometz/Art-Buddy"
            rel="noreferrer"
            target="_blank"
            title="Github repo for Art Buddy"
          >
            <svg
              aria-hidden="true"
              class="svg-inline--fa fa-github fa-w-16 "
              data-icon="github"
              data-prefix="fab"
              focusable="false"
              role="img"
              viewBox="0 0 496 512"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M165.9 397.4c0 2-2.3 3.6-5.2 3.6-3.3.3-5.6-1.3-5.6-3.6 0-2 2.3-3.6 5.2-3.6 3-.3 5.6 1.3 5.6 3.6zm-31.1-4.5c-.7 2 1.3 4.3 4.3 4.9 2.6 1 5.6 0 6.2-2s-1.3-4.3-4.3-5.2c-2.6-.7-5.5.3-6.2 2.3zm44.2-1.7c-2.9.7-4.9 2.6-4.6 4.9.3 2 2.9 3.3 5.9 2.6 2.9-.7 4.9-2.6 4.6-4.6-.3-1.9-3-3.2-5.9-2.9zM244.8 8C106.1 8 0 113.3 0 252c0 110.9 69.8 205.8 169.5 239.2 12.8 2.3 17.3-5.6 17.3-12.1 0-6.2-.3-40.4-.3-61.4 0 0-70 15-84.7-29.8 0 0-11.4-29.1-27.8-36.6 0 0-22.9-15.7 1.6-15.4 0 0 24.9 2 38.6 25.8 21.9 38.6 58.6 27.5 72.9 20.9 2.3-16 8.8-27.1 16-33.7-55.9-6.2-112.3-14.3-112.3-110.5 0-27.5 7.6-41.3 23.6-58.9-2.6-6.5-11.1-33.3 2.6-67.9 20.9-6.5 69 27 69 27 20-5.6 41.5-8.5 62.8-8.5s42.8 2.9 62.8 8.5c0 0 48.1-33.6 69-27 13.7 34.7 5.2 61.4 2.6 67.9 16 17.7 25.8 31.5 25.8 58.9 0 96.5-58.9 104.2-114.8 110.5 9.2 7.9 17 22.9 17 46.4 0 33.7-.3 75.4-.3 83.6 0 6.5 4.6 14.4 17.3 12.1C428.2 457.8 496 362.9 496 252 496 113.3 383.5 8 244.8 8zM97.2 352.9c-1.3 1-1 3.3.7 5.2 1.6 1.6 3.9 2.3 5.2 1 1.3-1 1-3.3-.7-5.2-1.6-1.6-3.9-2.3-5.2-1zm-10.8-8.1c-.7 1.3.3 2.9 2.3 3.9 1.6 1 3.6.7 4.3-.7.7-1.3-.3-2.9-2.3-3.9-2-.6-3.6-.3-4.3.7zm32.4 35.6c-1.6 1.3-1 4.3 1.3 6.2 2.3 2.3 5.2 2.6 6.5 1 1.3-1.3.7-4.3-1.3-6.2-2.2-2.3-5.2-2.6-6.5-1zm-11.4-14.7c-1.6 1-1.6 3.6 0 5.9 1.6 2.3 4.3 3.3 5.6 2.3 1.6-1.3 1.6-3.9 0-6.2-1.4-2.3-4-3.3-5.6-2z"
                fill="currentColor"
              />
            </svg>
          </a>
        </div>
      </div>
    </div>
  </section>
</DocumentFragment>
`;
